gprofiler/client.py

```diff
@@ -63,7 +63,7 @@ def _send_request(
         self,
         method: str,
         path: str,
-        data: Optional[Dict],
+        data: Any,
```

Comment:
Why? Is it really not always a dict?

Reply:
yes :(

From `log.py:191`, a list is passed:
```python
            self._api_client.post(self._path, data=self._logs[:logs_count], api_version="v1")
```

Reply:
Oof